README.md

```diff
@@ -12,8 +12,8 @@ You can install the `mkdocs_autodoc` package using `pip` or `easy_install`
 Dynamic Generation
 --------------
 
-To have the content always generate dynamically, without the use of any markdown files, you can use the
-`autodoc` page command.  This will allow you to dynamically insert a python module into the page heirarchy
+To have the content always generated dynamically, without the use of any markdown files, you can use the
```

Comment:
I think 'generate' makes more sense, however the full thing could perhaps be re-worded.